recipes/phelel/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "phelel" %}
+{% set version = "0.6.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/phelel-{{ version }}.tar.gz
+  sha256: 075f1ae0621c69f327c6739e3759e681735135af69f57b965376552a6907afa2
+
+build:
+  entry_points:
+    - phelel = phelel.scripts.phelel:run
+    - phelel-load = phelel.scripts.phelel_load:run
+    - velph = phelel.velph.cli:cmd_root
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: true  # [win]
```

Comment:
Looks like this one can ve noarch, no?

It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.

Reply:
Thank you very much for your prompt response. Yes, it can be `noarch: python`. Initially, due to the Windows check failure, I considered skipping Windows as a solution.
Based on your feedback, I have reverted to `noarch: python`. I would appreciate it if you could review this change.